lib/services/groups/groupRegistryMongoDB.js

```diff
@@ -180,6 +219,36 @@ function getById(id, callback) {
     });
 }
 
+/**
+ * List all the configGroups created in the IoT Agent for a service and a subservice.
+ *
+ * @param  {String} service        Service used to filter the groups.
+ * @param  {String} subservice     Subservice used to filter the groups.
+ * @param  {Function} callback     The callback function.
+ */
+function findConfigGroups(service, subservice, callback) {
```

Comment:
This function seems to be the same than `find()`. Could you elaborate on the differences, please?

Reply:
In the `find()` function, the callback function stores the result into `services` element while in `findConfigGroups()` function the callback function stores the result into `configGroups` element. It is difficult to change the returned element on fly mode as it is not known which endpoint calls the `find()` inside the function. So, I have created two different functions for /iot/services endpoint and /iot/configGroups endpoint. If my understanding is not correct, please share your opinion.